client/src/pages/private/SiteTable.js

```diff
@@ -15,7 +15,9 @@ const columns = [
   { id: 'operatorName', name: 'Operator Name' },
   { id: 'healthAuthority', name: 'Health Authority' },
   { id: 'postalCode', name: 'Postal Code' },
+  { id: 'details', name: 'Details' },
```

Comment:
I think for our other tables we didn't name this column - is this going to show up as blank? IMO we should match the style we're using elsewhere, if this doesn't already :)

Reply:
Yep, we made it so you can omit the `name` property and the column title will be blank.